source/Cosmos.HAL2/PCSpeaker.cs

```diff
@@ -85,9 +85,13 @@ public static void Beep(uint frequency)
         /// <exception cref="ArgumentOutOfRangeException">Thrown if duration or frequency invalid.</exception>
         public static void Beep(uint frequency, uint duration)
         {
+            if (frequency < 37 || frequency > 32767)
+            {
+                throw new ArgumentOutOfRangeException("'frequency' must be between 37 and 32767Hz.");
```

Comment:
The correct way to do that would be `throw new ArgumentOutOfRangeException("frequency", "Value must be between 37 and 32767");` as the first constructor argument is paramName and the second one is message

Reply:
> The correct way to do that would be `throw new ArgumentOutOfRangeException("frequency", "Value must be between 37 and 32767");` as the first constructor argument is paramName and the second one is message

Opposite of that, but thanks

Reply:
> Opposite of that, but thanks

The signature for the `ArgumentOutOfRangeException` constructor is `public ArgumentOutOfRangeException (string? paramName, string? message);`. I'd recommend using `throw new ArgumentOutOfRangeException(nameof(frequency), "Value must be between 37 and 32767");`, since that's the standard way to throw such exceptions, though.